lib/presentation/util/app_image_paths.dart

```diff
@@ -49,6 +49,7 @@ class AppImagePaths {
   String get icFileTypeAudio => _getImagePath('ic_file_type_audio.svg');
   String get icLinShareMenu => _getImagePath('ic_linshare_menu.svg');
   String get icLinShareLogo => _getImagePath('ic_linshare_logo.svg');
+  String get icError => _getImagePath('ic_unexpected_error.svg');
```

Comment:
Please also change the string name to `icUnexpectedError` :(

Reply:
:(